test/comparison/go/relations/1group.obx.go.expected

```diff
@@ -98,7 +98,7 @@ func (group_EntityInfo) Load(ob *objectbox.ObjectBox, bytes []byte) (interface{}
 
 	return &Group{
 		Id:   propId,
-		Name: fbutils.GetStringSlot(table, 6),
+		Name: fbutils.GetStringSlot(table, 2),
```

Comment:
and this one... actually not going to comment on the rest... I'll just restore all the accidentally changed "expected files"